dpctl/_sycl_device.pyx

```diff
@@ -639,6 +643,22 @@ cdef class SyclDevice(_SyclDevice):
             self._device_ref
         )
 
+    @property
+    def platform(self):
+        """ Returns the platform associated with this device.
+
+        Returns:
+            :class:`dpctl.SyclPlatform`: The platform associated with this
+                device.
+        """
+        cdef DPCTLSyclPlatformRef PRef = (
+            DPCTLDevice_GetPlatform(self._device_ref)
+            )
```

Comment:
indent is off. Should line up with the `cdef`

Reply:
Fixed